device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceResourceSpec.scala

```diff
@@ -198,52 +176,6 @@ class DeviceResourceSpec extends ResourcePropSpec {
     }
   }
 
-  property("First POST request on 'ping' should update 'activatedAt' field for device.") {
```

Comment:
These tests still makes sense, but instead of calling `devicePing` we could call the action in the `DeviceSeenListener`. Maybe it makes sense to move these testss.